cashu/core/crypto/dlc.py

```diff
@@ -0,0 +1,59 @@
+from hashlib import sha256
+from typing import List, Optional, Tuple
+
+
+def sorted_merkle_hash(left: bytes, right: bytes) -> bytes:
+    '''Sorts `left` and `right` in non-ascending order and
+        computes the hash of their concatenation
+    '''
+    if int.from_bytes(left, 'big') < int.from_bytes(right, 'big'):
+        left, right = right, left
```

Comment:
FWIW, you can simplify to:

```suggestion
    if left < right:
        left, right = right, left
```

Also is there any reason you decided to swap the sort order from the spec? The spec says to sort branch hashes in ascending order, but you're doing descending order here.

Reply:
Will do that, thanks.